docs/models/cx_collections.md

```diff
@@ -17,7 +17,6 @@ Data is published for download at https://www.performance.gov/cx/data/.
 ++| organization_name (Service provider    | integer  | Organization Name                                                           | General Services Administration       |
 | quarter                                  | string   | The quarter of the fiscal year                                              | 2                                     |
 | rating                                   | string   | The rating for the service                                                  | true,false,partial                    |
-| service_id                               | integer  | Identifier for the service                                                  | 74                                    |
```

Comment:
I'm surprised this is being removed from `cx_collections.md` rather than from `cx_collection_details.md`

Reply:
good catch here. the docs should be cx collection details indeed.
will update.